### DISCUSSION TITLE
ENOENT error no such file or directory,

### DISCUSSION THREAD

**Discussion by gewoondemax** [advplyr/audiobookshelf#1753] [Q&A]
Hi all.

brand new to advplyr audio bookshelf running on Docker MacOS Ventura MacbookPro i7 16Gb/245GB SSD

In the log files I see error with this message ..

ERROR
[Watcher] Error: ENOENT: no such file or directory, watch '/audiobooks/_AUDIOBOOKSHELF/libraryItems/data/data.0.json.lock'
ERROR
[Watcher] Error: ENOENT: no such file or directory, watch '/audiobooks/_AUDIOBOOKSHELF/libraryItems/data/data.1.json.lock'
ERROR
[Watcher] Error: ENOENT: no such file or directory, watch '/audiobooks/_AUDIOBOOKSHELF/libraryItems/data/data.2.json.lock'

uo to data.4.json.lock

what is this and what can I change to or should change to ..?

**Comment by advplyr** [comment]
It looks like you have put your `/config` folder inside one of your library folders.
Make sure the `/config` and `/metadata` folders are not contained in each other and not contained in a library folder.

**Reply by gewoondemax** [reply]
Thank you please can you advise where can change the paths? is this in the dashboard or via terminal?

**Reply by advplyr** [reply]
You specify the paths when setting up Docker. https://www.audiobookshelf.org/docs#docker-install
Someone can probably help you in Discord/Matrix

**Comment by gewoondemax** [comment]
Thank you for the info - 

Indeed you were right - just by adding the missing directories and made sure they were read and writeable solved the warning/ error.
I must say what an sublime work you created!!!!!

It is flawless how the connection works between app and audiobookshelf. Switching from Mesh WiFi to 4G 5G it works perfect.
Docker 4.49 runs on a Mac Mini 2011 i7 16Gb/500GB raid 0 with Ventura 13.13a installed using Legacy Core Patcher.
I followed the Windows installation :-) in the documentation (perfect)

====

One, maybe two question …

How / where to change the time zone setting? Because it runs 2 hours late … :-(
Also how to change the language … I put the server on NL but my iPhone is still EN … restarted all, but nothing (server version is NL but Docker is EN)

How to add / change the Authors biography I thing the Audnex is now bases on the us market? …. But can this be changed to world wide or maybe different country?

=====

Thank you again for the great work!!!!

Max

**Reply by advplyr** [reply]
You set the timezone in your Docker run command.

Translations haven't been added to the mobile apps yet.

If you set the book provider for the library to Audible for the country you want then the Author lookup will respect that. Keep in mind this uses Audnexus so issues with results would be outside of our control.

**Comment by gewoondemax** [comment]
Thank you! Outstanding ! Another question. Is there an easy way to run Audiobookshelf on https? If you look @ Calibre they have integrate the possibility to add privat key and certificate.Maybe feature request?Thank you 1000 times Regards MaxVerstuurd vanaf mijn iPhoneOp 11 mei 2023 om 20:55 heeft advplyr ***@***.***> het volgende geschreven:﻿
You set the timezone in your Docker run command.
Translations haven't been added to the mobile apps yet.
If you set the book provider for the library to Audible for the country you want then the Author lookup will respect that. Keep in mind this uses Audnexus so issues with results would be outside of our control.

**Reply by advplyr** [reply]
The way to run Abs over https is to setup a reverse proxy. I'm not sure about having a place to add certificate I've never done that before.
